ftl/core/browsing.ftl

```diff
@@ -83,6 +83,7 @@ browsing-reposition = Reposition...
 browsing-reposition-new-cards = Reposition New Cards
 browsing-reschedule = Reschedule
 browsing-save-current-filter = Save Current Filter...
+browsing-search-bar-hint = Search cards/notes
```

Comment:
I was actually trying to be verbose because the search bar is so long and I thought less empty space looked better. That might not be a good enough reason, but I also think it we could provide more helpful text. For example: "Type here and press Enter to search for cards/notes."

Reply:
Yeah, I think I went too far. I was also thinking it might be nice to remove the Search button to add some more room, and to do that we really do need to mention pressing enter.

Reply:
That's a cool idea. I leave the hint as it is then and it can be updated accordingly with the GUI change.